test/support/helpers/context_dir.js

```diff
@@ -123,7 +123,7 @@ class ContextDir {
   }
 
   async move (src /*: string */, dst /*: string */) {
-    return fs.move(this.abspath(src), this.abspath(dst))
+    return fs.rename(this.abspath(src), this.abspath(dst))
```

Comment:
What's the difference between those 2?

Reply:
[`Local#_move()`](https://github.com/cozy-labs/cozy-desktop/blob/v3.11.1-beta.1/core/local/index.js#L342) actually uses [`fs.rename()`](https://github.com/cozy-labs/cozy-desktop/blob/v3.11.1-beta.1/core/local/index.js#L359), so this better matches our own sync actions.

Also `fs.move` actually comes from `fs-extra`, not `fs`, it does some black magic to ensure move succeeds in weird cases (e.g. separate volumes), and may end up changing the inode of watched files.